ArcGISRuntimeSDKQt_CppSamples/Routing/NavigateARouteWithRerouting/NavigateARouteWithRerouting.cpp

```diff
@@ -294,11 +293,13 @@ void NavigateARouteWithRerouting::connectRouteTrackerSignals()
         textString += "Time remaining: " + time.toString("hh:mm:ss") + "\n";
 
         // display next direction
-        if (DirectionManeuverListModel* directionList = dynamic_cast<DirectionManeuverListModel*>(m_directions))
+        m_directionManeuvers.clear();
+        m_directionManeuvers = m_route.directionManeuvers(this)->directionManeuvers();
```

Comment:
Add these lines only when RouteTracker returns `rerouteCompleted`

Reply:
Maybe try adding it somewhere in [this](https://github.com/Esri/arcgis-runtime-samples-qt/blob/6e5fe638d9f0b810f45273e6ca96a7c3a25cd6a7/ArcGISRuntimeSDKQt_CppSamples/Routing/NavigateARouteWithRerouting/NavigateARouteWithRerouting.cpp#L253) method.